File: popfragg.Domain/DTOS/Authorizer/Responses/User/UserResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace popfragg.Domain.DTOS.Authorizer.Responses.User
{
    public class UserResponse
    {
        public string Id { get; set; } = default!;
        public string Email { get; set; } = default!;
        public bool Email_Verified { get; set; }
        public string Given_Name { get; set; } = default!;
        public string Family_Name { get; set; } = default!;
        public string Nickname { get; set; } = default!;
        public string Preferred_Username { get; set; } = default!;
        public string Signup_Methods { get; set; } = default!;
        public bool Phone_Number_Verified { get; set; }
        public List<string> Roles { get; set; } = [];
        public long Created_At { get; set; }
        public long Updated_At { get; set; }
        public AppDataResponse App_Data { get; set; } = default!;
    }
}

File: popfragg.Tools.Codegen/Outputs/schemas/user-response.ts
import { z } from 'zod';

export const userResponseSchema = z.object({
  id: z.string().optional(),
  email: z.string().optional(),
  email_Verified: z.boolean(),
  given_Name: z.string().optional(),
  family_Name: z.string().optional(),
  nickname: z.string().optional(),
  preferred_Username: z.string().optional(),
  signup_Methods: z.string().optional(),
  phone_Number_Verified: z.boolean(),
  roles: z.array(z.string()),
  created_At: z.number(),
  updated_At: z.number(),
  app_Data: z.any().optional(),
});

export type UserResponse = z.infer<typeof userResponseSchema>;
